diff --git a/CMakeLists.txt b/CMakeLists.txt
index 94b10d5..81a33b0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,6 +126,13 @@ endif()
 
 add_subdirectory(src)
 
-if((IS_MAIN_PROJECT OR argp-standalone_BUILD_TESTING) AND BUILD_TESTING)
-  add_subdirectory(test)
-endif()
+include(GNUInstallDirs)
+
+install(TARGETS argp-standalone
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+install(FILES
+  ${PROJECT_SOURCE_DIR}/include/argp-standalone/argp.h
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
